recipes/xtensor-blas/0.17.2/conanfile.py

```diff
@@ -0,0 +1,89 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import textwrap
+
+required_conan_version = ">=1.43.0"
+
+
+class XtensorBlasConan(ConanFile):
+    name = "xtensor-blas"
+    description = "Extension to the xtensor library, offering bindings to BLAS and LAPACK libraries through cxxblas and cxxlapack from the FLENS project."
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/xtensor-stack/xtensor-blas"
+    topics = ("numpy", "multidimensional-arrays",
+              "tensors", "openblas", "blas", "lapack")
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("xtensor/0.21.4")
+        self.requires("openblas/0.3.17")
+
+    def validate(self):
+        # https://github.com/xtensor-stack/xtensor/blob/master/README.md
+        # - On Windows platforms, Visual C++ 2015 Update 2, or more recent
+        # - On Unix platforms, gcc 4.9 or a recent version of Clang
+        version = tools.Version(self.settings.compiler.version)
+        compiler = self.settings.compiler
+        if compiler == "Visual Studio" and version < "16":
+            raise ConanInvalidConfiguration(
+                "xtensor requires at least Visual Studio version 15.9, please use 16"
+            )
+        if (compiler == "gcc" and version < "5.0") or (
+            compiler == "clang" and version < "4"
+        ):
+            raise ConanInvalidConfiguration("xtensor requires at least C++14")
+
+    def configure(self):
+        self.options["openblas"].build_lapack = True
```

Comment:
Unfortunately, this option is `False` by default and Conan Center does not produce it as `True`.

Reply:
It's my first time to write a conanfile, is there another way to change the default option? Thank you.

Reply:
The question is, is it possible to build xtensor without that option?

Reply:
> Need to check openblas project. What they use by default, and what does happen when build lapack is True to other projects. IIRC lapack package is broken, because it requires gfortran and fortran conan package is broken.

When building xtensor-blas locally, I just installed gfortran, and set `build_lapack=True`, no requirement for fortran conan package I think.

Reply:
@xhuohai was you able to build xtensor with lapack conan package? Asking because I'm not sure if it's working

Reply:
Question is I can not find lapack conan package at connacenter or conan-bintray.

Reply:
https://github.com/conan-community/conan-lapack says lapack is on Bintray, but I can't get access to it.

*ERROR: Failed requirement 'lapack/3.7.1' from 'xtensor-blas/0.17.2'
ERROR: HTTPSConnectionPool(host='conan.bintray.com', port=443): Max retries exceeded with url: /v1/ping (Caused by SSLError(SSLError(1, '[SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1131)')))
Unable to connect to conan-center=https://conan.bintray.com*

Reply:
conan-bintray is deprecated and well documented: https://docs.conan.io/en/latest/faq/troubleshooting.html#error-httpsconnectionpool-host-conan-bintray-com-port-443

conancenter is the only official remote.

remove bintray `conan remote remove conan-bintray`

Reply:
I removed bintray. But lapack is not contained in conancenter or other remote I added before.
![image](https://user-images.githubusercontent.com/23413563/174283231-0f53f7c6-ad1a-4d5e-aa7c-93b35c494cb7.png)

Reply:
Indeed it has been removed: `conan search -r conancenter lapack`. It's because that package was broken and really hard to be fixed. The possible option is fix lapack to continue this PR.